hash/msx2_flop.xml

```diff
@@ -15471,6 +15748,30 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="yoshikmd2">
+		<description>Yoshida Koumuten Data Library Vol. 2 (Japan)</description>
+		<year>1990</year>
+		<publisher>MSX Magazine</publisher>
+		<info name="alt_title" value="吉田工務店データ集全３巻 VOL.2"/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="yoshida koumuten data library vol. 2 - msx magazine.dsk" size="737280" crc="559c3ce2" sha1="bc83532e77326bdecbe1818f3c15b5f5de572a45"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="yoshikmd3">
+		<description>Yoshida Koumuten Data Library Vol. 3 (Japan)</description>
+		<year>1990</year>
+		<publisher>MSX Magazine</publisher>
+		<info name="alt_title" value="吉田工務店データ集全３巻 VOL.3"/>
```

Comment:
Same for this one – where does the title come from?

Reply:
Also from generation-msx. Updated to match the name displayed on the title screen.